controllers/grid/users/stageParticipant/form/PKPStageParticipantNotifyForm.inc.php

```diff
@@ -181,7 +181,7 @@ function sendMessage($userId, $submission, $request) {
 
 			$suppressNotificationEmail = false;
 
-			if (!$email->send($request)) {
+			if ($email->isEnabled() && !$email->send($request)) {
```

Comment:
The request is optional, but used for logging, so I'll leave it there.
Initially I had changed a lot the code, to avoid generating the email if it's disabled, but I reverted, to avoid adding issues, perhaps I'll be more aggressive on the `main` branch.